### ISSUE TITLE
Display title module for wiki pages

### ISSUE THREAD

**Issue by Randomno** [TASVideos/tasvideos#1612]
Regardless of #1611 this is a useful tool.

Example usage: `[module:displaytitle|SDA runs without a published TAS]`

Result:

![image](https://github.com/TASVideos/tasvideos/assets/17330834/0a7102f5-4e45-4edf-876e-d39c113b671c)

Compare MediaWiki's `{{DISPLAYTITLE}}`: https://www.mediawiki.org/wiki/Extension:Display_Title

**Comment by vadosnaprimer** [comment]
I like this much more.

**Comment by nattthebear** [comment]
No, this is not good.  If we want the presentation name of the article to be separate from the actual name, we need to store it on the entity and not buried in the markup.

**Comment by YoshiRulz** [comment]
On a related note, could we have a way to set `<link rel="canonical"/>`? It must appear in the `<head/>`.
